apps/survey/contracts/Survey.sol

```diff
@@ -204,21 +225,22 @@ contract Survey is AragonApp {
         return surveys[_surveyId].metadata;
     }
 
-    function getVoterState(uint256 _surveyId, address _voter) public view returns (uint256[] options, uint256[] stakes) {
-        if (surveys[_surveyId].voters[_voter].optionsCastedLength == 0) {
+    function getVoterState(uint256 _surveyId, address _voter) external view returns (uint256[] options, uint256[] stakes) {
+        if (surveys[_surveyId].votes[_voter].optionsCastedLength == 0) {
             return (new uint256[](0), new uint256[](0));
         }
 
-        options = new uint256[](surveys[_surveyId].voters[_voter].optionsCastedLength + 1);
-        stakes = new uint256[](surveys[_surveyId].voters[_voter].optionsCastedLength + 1);
-        for (uint256 i = 0; i <= surveys[_surveyId].voters[_voter].optionsCastedLength; i++) {
-            options[i] = surveys[_surveyId].voters[_voter].castedVotes[i].optionId;
-            stakes[i] = surveys[_surveyId].voters[_voter].castedVotes[i].stake;
+        MultiOptionVote storage vote = surveys[_surveyId].votes[_voter];
+        options = new uint256[](vote.optionsCastedLength + 1);
+        stakes = new uint256[](vote.optionsCastedLength + 1);
+        for (uint256 i = 0; i <= vote.optionsCastedLength; i++) {
+            options[i] = vote.castedVotes[i].optionId;
+            stakes[i] = vote.castedVotes[i].stake;
         }
     }
 
-    function getOptionSupport(uint256 _surveyId, uint256 _option) public view returns (uint256) {
-        return surveys[_surveyId].votes[_option];
+    function getOptionSupport(uint256 _surveyId, uint256 _optionId) public view returns (uint256) {
+        return surveys[_surveyId].optionVotes[_optionId];
     }
 
     function isParticipationAchieved(uint256 _surveyId) public view returns (bool) {
```

Comment:
I'd rename `survey.minAcceptParticipationPct` to just `survey.minParticipationPct`, since there's no real "acceptance".